### DISCUSSION TITLE
Any (other) offline visualization tools for SBFspot?

### DISCUSSION THREAD

**Discussion by philipptrenz** [SBFspot/SBFspot#463] [General]
Hi guys, many thanks for your excellent work! 

I was searching for a offline visualization tool and found https://github.com/KiOui/SunPortal
Based on the work of @KiOui I'm working on more features [here](https://github.com/philipptrenz/sunportal/). I wanted to share this and also ask if there are any other offline visualization tools for @SBFspot I've overlooked?

**Comment by mincequi** [comment]
I am working on an App (iOS, Android) to support visualization of SBFspot data.
Initial version is working and displays live data only (historic data visualization is in the working).
See screenshot and let me know, if someone is interested.

![Screenshot 2021-02-16 at 17 42 17](https://user-images.githubusercontent.com/1805183/108094225-dfed6780-707e-11eb-8d8e-155d45042c65.png)

**Comment by aenne23769** [comment]
Very interested

**Comment by sbf-** [comment]
Me too

**Comment by sneumeister** [comment]
For visualization I use 123solar:
https://www.123solar.org/

Runs on same Pi 2B as SBFSpot.
Just modified to feed data from SBFSpot sqlite-database (not via "-123s" option).

**Comment by shadowbyte68** [comment]
Try GRAFANA !
